virtcontainers/kata_agent.go

```diff
@@ -719,6 +719,30 @@ func (k *kataAgent) replaceOCIMountSource(spec *specs.Spec, guestMounts []Mount)
 	return nil
 }
 
+func (k *kataAgent) removeIgnoredOCIMount(spec *specs.Spec, ignoredMounts []Mount) error {
+	var mounts []specs.Mount
+
+	for _, m := range spec.Mounts {
+		found := false
+		for _, ignoredMount := range ignoredMounts {
+			if ignoredMount.Source == m.Source {
+				k.Logger().Debugf("Removing OCI mount (%s)", m.Source)
```

Comment:
Nit: this one could be converted to a structured format too.

Reply:
ooooooh.  ok, fixed!

Reply:
Ta! ;)